cfgrammar/src/lib/yacc/parser.rs

```diff
@@ -67,6 +68,9 @@ impl fmt::Display for YaccParserError {
             YaccParserErrorKind::UnknownDeclaration => "Unknown declaration",
             YaccParserErrorKind::DuplicatePrecedence => "Token already has a precedence",
             YaccParserErrorKind::PrecNotFollowedByToken => "%prec not followed by token name",
+            YaccParserErrorKind::DuplicateAvoidInsertDeclaration => {
+                "Duplicate %avoid_insert declaration"
```

Comment:
Do we need to disallow multiple `%avoid_insert`s? Given it's a HashSet anyways, we could just merge them all together, similar to the `%token` declarations. This gives users more options to format their grammars (e.g. if they have loads of tokens they want to avoid, they don't need to be all in a single line), and has as far as I can tell no disadvantage.

Reply:
I sort-of agree, but `%implicit_tokens` has the same restriction. Perhaps we should loosen both?

Reply:
I don't see a reason why not. But we can do this in separate PR.

Reply:
Agreed, that sounds like a plan. Let's do both in one go?